transformer_engine/jax/csrc/modules.cpp

```diff
@@ -786,8 +797,9 @@ void RMSNormBackward(cudaStream_t stream, void **buffers, const char *opaque, si
     auto hidden_size = desc.hidden_size;
     auto wkspace_size = desc.wkspace_size;
     auto barrier_size = desc.barrier_size;
-    auto dgamma_part_sizes = desc.dgamma_part_sizes;
-    size_t dbeta_part_sizes[2] = {0, 0};
+    auto dgamma_part_shape = desc.dgamma_part_shape;
+    Shape dbeta_part_shape;
```

Comment:
The `std::vector` shapes coming in from Python are being converted to `Shape` structs only to be converted back to an `std::vector` when creating the `TensorWrapper`s for these tensors. Should we just keep them as `std::vector` in the descriptor?

Reply:
The current [packing method](https://github.com/mingxu1067/TransformerEngine/blob/mingh/fix_wrong_dgamma_shape_ln_bwd/transformer_engine/jax/csrc/modules.cpp#L40) does not support to pack dynamic data pointers, therefore putting `std::vector` to descriptors would led to miss few information when unpacking. `Shape` is a structure to contains fixed length array to be compatiable to packing method.